ega-charts/cega/templates/cega-deploy.yaml

```diff
@@ -14,7 +14,13 @@ spec:
       labels:
         app: cega-users
         role: fake-users
+      annotations:
+        checksum/secret: {{ include (print $.Template.BasePath "/cega-secret.yaml") . | sha256sum }}
     spec:
+      securityContext:
+        runAsUser: 1000
```

Comment:
This will work on Debian and Linux distros. Others such as RedHat start non-privileged users with 500, for example.

Reply:
What this does is to set the runtime user for the k8s pod to the specified UID, as such any UID > 0 will be classified as unprivileged by the RBAC system.

Reply:
Good!

Reply:
I think it is highly reccomendable to set allowPrivilegeEscalation flag to False. This way we will ensure that no child process of a container can gain more privileges than its parent. Read more: https://kubernetes.io/docs/concepts/policy/pod-security-policy/#users-and-groups

Reply:
It is set in the underlying k8s config.